python/pyspark/sql/types.py

```diff
@@ -1249,7 +1249,7 @@ def _infer_schema_type(obj, dataType):
 }
 
 
-def _verify_type(obj, dataType, nullable=True):
+def _verify_type(obj, dataType, nullable=True, name="obj"):
```

Comment:
This will print "obj" when called from `session.createDataFrame` (https://github.com/dgingrich/spark/blob/topic-spark-19507-verify-types/python/pyspark/sql/session.py#L408).  It'd be easy to set the name where it's called but it wasn't clear what to set it to.   The input can be either an RDD, list, or pandas.DataFrame.